<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPC_Partners_manager</class>
 <widget class="QDialog" name="CPC_Partners_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPC_Partners_manager</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>931</width>
     <height>751</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>수집</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>341</height>
      </rect>
     </property>
     <property name="title">
      <string>1. 수집할 카테고리 선택</string>
     </property>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>211</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="story_checkBox">
           <property name="text">
            <string>스토리</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="racy_joke_checkBox">
           <property name="text">
            <string>응큼세포(15금)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="entertainment_hot_issue_checkBox">
           <property name="text">
            <string>연예가화제</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="idol_checkBox">
           <property name="text">
            <string>아이돌</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="economy_checkBox">
           <property name="text">
            <string>경제</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="tip_checkBox">
           <property name="text">
            <string>꿀팁</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="global_checkBox">
           <property name="text">
            <string>글로벌</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="entertainment_checkBox">
           <property name="text">
            <string>연예</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="overseas_soccer_checkBox">
           <property name="text">
            <string>해외축구</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="life_pick_checkBox">
           <property name="text">
            <string>생활픽</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BBC_News_checkBox">
           <property name="text">
            <string>BBC News</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QCheckBox" name="joke_checkBox">
           <property name="text">
            <string>유머</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="politics_checkBox">
           <property name="text">
            <string>정치</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="accident_checkBox">
           <property name="text">
            <string>사건사고</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sport_checkBox">
           <property name="text">
            <string>스포츠</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="vehicle_checkBox">
           <property name="text">
            <string>자동차</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="society_checkBox">
           <property name="text">
            <string>사회</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pet_checkBox">
           <property name="text">
            <string>반려동물</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="overseas_baseball_checkBox">
           <property name="text">
            <string>해외야구</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="travel_checkBox">
           <property name="text">
            <string>여행</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="NNA_Korea_checkBox">
           <property name="text">
            <string>NNA코리아</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QRadioButton" name="collect_category_radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>개별 카테고리 수집</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="collect_main_radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>메인 페이지 (전체 카테고리)</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>2. 수집 개수 설정</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLineEdit" name="item_amount_edit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>개의 게시물을 수집</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="crawling_srt_btn">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>22</y>
       <width>75</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string>수집 시작</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>정책설정</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>수집된 뉴스피드 관리 및 업로드</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>901</width>
       <height>441</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>899</width>
        <height>439</height>
       </rect>
      </property>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>201</width>
         <height>281</height>
        </rect>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QLabel" name="test_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>111</width>
          <height>111</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>181</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>제목</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>게시일</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>수집일</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>배포일</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>출처</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>상세URL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>공유URL</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QLabel" name="title_label">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="posting_date_label">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>         </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="collect_date_label">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>         </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="distribution_date_label">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="source_label">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="detail_URL_label">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>          </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="Share_URL_label">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>          </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>아이디 설정</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>뉴스픽 로그인</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>124</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>  ID   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="News_Pick_ID_edit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>  PW  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="News_Pick_PW_edit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>유튜브 로그인</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>124</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>  ID   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Utube_ID_edit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>  PW  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Utube_PW_edit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>오아시스 로그인</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>124</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>  ID   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Oasis_ID_edit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>  PW  </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Oasis_PW_edit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>라이센스</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
